z_infra_provisioning/cloud_infra_center/ocp_upi/quick-install.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+############################################################################
+#  Author:  Ling Ling Hu
+#  Email: shhull@cn.ibm.com
+#  Description: This script is used to run ansible script more quick in a brand new RHEL environment. 
+#  Please do not use it in any CIC nodes and your older-machine.
+#  ONLY new VM or RHEL machine
+############################################################################
+
+# ensure running as root
+if [ "$(id -u)" != "0" ]; then
+  exec sudo "$0" "$@"
+fi
+
+if [ $# -ne 3 ]; then
+  echo 'This script requires 3 parameters:'
+  echo '  CONTROL_NODES - Which CIC control node you want to deploy OCP.'
+  echo '  USERNAME - RedHat register username.'
+  echo '  PASSWORD - Redhat register password.'
+  exit 1
+fi
+
+CONTROL_NODES="${1}"
+USERNAME="${2}"
+PASSWORD="${3}"
+CIC_USER="root"
+CIC_PWD="dflatpass"
+
+redhat_register() {
+    sudo subscription-manager register --username ${USERNAME} --password ${PASSWORD} --auto-attach --force
+    sudo subscription-manager attach
+    sudo subscription-manager repos --enable=ansible-2.8-for-rhel-8-s390x-rpms 
+}
+
+package_install() {
+    sudo yum install sshpass git firewalld -y
+    sudo yum install python3 ansible jq wget -y
+    sudo -H pip3 install --upgrade pip
+    sudo dnf update -y
+    sudo dnf install redhat-rpm-config gcc libffi-devel python3-devel openssl-devel cargo -y
+cat <<'EOF' >> requirements.txt 
+# The order of packages is significant, because pip processes them in the order
+# of appearance. Changing the order has an impact on the overall integration
+# process, which may cause wedges in the gate later.
+pbr!=2.1.0,>=2.0.0 # Apache-2.0
+
+cliff>=3.5.0 # Apache-2.0
+iso8601>=0.1.11 # MIT
+openstacksdk>=0.57.0 # Apache-2.0
+osc-lib>=2.3.0 # Apache-2.0
+oslo.i18n>=3.15.3 # Apache-2.0
+oslo.utils>=3.33.0 # Apache-2.0
+python-keystoneclient>=3.22.0 # Apache-2.0
+python-novaclient>=17.0.0 # Apache-2.0
+python-cinderclient>=3.3.0 # Apache-2.0
+stevedore>=2.0.1 # Apache-2.0
+EOF
+    sudo pip3 install -r requirements.txt python-openstackclient --ignore-installed
+    sudo alternatives --set python /usr/bin/python3
+}
+
+prepare_install() {
+# Default CIC password is dfltpass
+cat <<'EOF' >> password.txt 
+dfltpass
```

Comment:
The default password of ICIC is the root user password of mgt node, not dfltpass.
How about make the password as a argument?

Reply:
yes, I have removed it.